lib/src/base_client.dart

```diff
@@ -42,8 +42,10 @@ abstract class BaseClient implements Client {
       _sendUnstreamed('PATCH', url, headers, body, encoding);
 
   @override
-  Future<Response> delete(url, {Map<String, String> headers}) =>
-      _sendUnstreamed('DELETE', url, headers);
+  Future<Response> delete(url,
+          {Map<String, String> headers, body, Encoding encoding}) =>
+      _sendUnstreamed('DELETE', url, headers, encoding);
```

Comment:
the `body` should also be forwarded to `_sendUnstreamed`. The `Client` abstract class should also be updated to include the updated method, and other classes implementing `Client` would probably need to be checked to be  updated to ensure `body` (and `encoding`) parameters passed are actually forwarded to `Request` method.

Reply:
Sorry, somehow missed that in the rebase. It should be fixed now.